File: 프로그래머스/0/120894. 영어가 싫어요/영어가 싫어요.js
function solution(numbers) {
    let numStr = [ "zero", "one", "two", "three", "four",
                  "five", "six", "seven", "eight", "nine" ];

    numStr.forEach((str, idx) => {
        numbers = numbers.replaceAll(str, idx);
    });
    return Number(numbers);
}

File: 프로그래머스/0/120894. 영어가 싫어요/영어가 싫어요.py
def solution(numbers):
    num_str_map = [
        "zero", "one", "two", "three", "four",
        "five", "six", "seven", "eight", "nine"
    ]
    for idx, str_word in enumerate(num_str_map):
        numbers = numbers.replace(str_word, str(idx))
    return int(numbers)